schema/types/app.go

```diff
@@ -28,6 +28,7 @@ type App struct {
 	Group             string         `json:"group"`
 	SupplementaryGIDs []int          `json:"supplementaryGIDs,omitempty"`
 	WorkingDirectory  string         `json:"workingDirectory,omitempty"`
+	ReadOnlyRootFS    bool           `json:"readOnlyRootFS,omitempty"`
```

Comment:
The idea is that it's a feature of the runtime rather than the image. So aci stays unchanged. Does that make sense?